### ISSUE TITLE
Site links are broken

### ISSUE THREAD

**Issue by tajmone** [Markdown2/site#3]
The website internal links are all broken: they have an extra  `site/` string in the URL which raises a 404 error page.

Manually removing  `site/` from the URL address in the browser show the correct page.

Eg:

- Link as found on site:
    ````
    http://markdown2.org/site/syntax/
    ````
- should instead be:
    ````
    http://markdown2.org/syntax/
    ````

Must be due to same changes in GitHub's Jekyll version.

**Comment by Serhioromano** [comment]
Yes. It have to be revised. Unfortunately right now I do not have time and there was not to much attention to this project.

But if I find other people who wants to make this move I'll grant some access.
